chargebee/src/main/java/com/chargebee/android/billingservice/BillingClientManager.kt

```diff
@@ -298,7 +298,7 @@ class BillingClientManager constructor(
     /* Chargebee method called here to validate receipt */
     private fun validateReceipt(purchaseToken: String, product: CBProduct) {
         try{
-        CBPurchase.validateReceipt(purchaseToken, customerID, product) {
+        CBPurchase.validateReceipt("purchaseToken", customerID, product) {
```

Comment:
`"purchaseToken"` looks like a mistake. @cb-haripriyan Did this slip through code review or what am I missing here?

Reply:
thanks @ciriousjoker for figuring out this hard coded values as have added for testing by mistake pushed without removing it.

Reply:
Updated the PR